kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternDetailsController.java

```diff
@@ -469,12 +469,19 @@ public void onChanged(Change<? extends DescrName> change) {
         StringBinding dateStrProp = Bindings.createStringBinding(
                 () -> {
                     if (patternProperty.get() != null) {
-                        return LocalDate.ofInstant(
-                                getViewProperties().calculator()
-                                        .getFullyQualifiedDescription(patternProperty.get().nid())
-                                        .get().instant(),
-                                ZoneId.systemDefault()
-                        ).format(DateTimeFormatter.ofPattern("MMM dd, yyyy"));
+                        long rawTime = getViewProperties().calculator()
+                                .getFullyQualifiedDescription(patternProperty.get().nid())
+                                .get().stamp().time();
+
+                        //Check if the timestamp is valid. If it matches Long.MIN_VALUE + 1
+                        if (rawTime == Long.MIN_VALUE + 1) {
+                            return "Premundane";
+                        }
+
+                        Instant stampInstance = Instant.ofEpochSecond(rawTime);
+                        ZonedDateTime stampTime = ZonedDateTime.ofInstant(stampInstance, ZoneOffset.UTC);
+                        String fqnDateAddedStr = DateTimeFormatter.ofPattern("MMM dd, yyyy").format(stampTime);
```

Comment:
Do we want to hard code the month first and then the day and then the year?
Some countries put the day first, and then the month and then the year... perhaps we should grab the current locale being used (could be spanish local, portuguse local, etc) and format it using that locale?

cc @carldea

Reply:
@dukke 
I worked with @bhharsh13  to format the date based on your local and now will format correctly.